src/vorta/utils.py

```diff
@@ -273,3 +273,19 @@ def is_system_tray_available():
         is_available = tray.isSystemTrayAvailable()
 
     return is_available
+
+
+def validate_passwords(firstPass, secondPass):
+    msg = "Passwords must be "
+    passEqual = firstPass == secondPass
+    passLong = len(firstPass) > 8
+    valid = passEqual and passLong
+
+    if not passEqual:
+        msg += "identical"
+    if not (passEqual or passLong):
+        msg += " and "
+    if not passLong:
+        msg += "greater than 8 characters long"
```

Comment:
that might work for english or languages with similar word order, but will not work for other translations.

you can't assemble from pieces when considering translation.

Reply:
Would this work for translation?
`

    if not passEqual:

        msg = "Passwords must be identical"

    if not passLong:

        msg = "Passwords must be greater than 8 characters long"

    if not (passLong or passEqual):

        msg = "Passwords must be identical and greater than 8 characters long"
`